bin/yaml/cpp.yaml

```diff
@@ -509,6 +509,28 @@ compilers:
             rm -Rf ~/.intel
             rm -Rf ~/intel
             rm -Rf /var/intel/installercache
+        - name: 2021.4.0.3201
+          fetch:
+            - https://registrationcenter-download.intel.com/akdlm/irc_nas/18209/l_dpcpp-cpp-compiler_p_{name}_offline.sh install.sh
+          script: |
+            rm -Rf ~/.intel
```

Comment:
We can probably de-duplicate some of this with yaml magic. I'll do so after merge :) Thanks!

Reply:
Thanks for the dedup. I did not know about that feature in yaml.